packages/Input/src/input.css

```
@@ -0,0 +1,137 @@
+@import "~@igloo-ui/tokens/dist/base10/variables.css";
+@import "~@igloo-ui/tokens/dist/fonts.css";
+
+:root {
+  /* Default */
+  --ids-input-height: var(--space-6);
+  --ids-input-font-family: var(--primary-font-family);
+  --ids-input-font-weight: var(--font-weight-regular);
+  --ids-input-padding: var(--space-3);
+  --ids-input-font-size: var(--font-size-4);
+  --ids-input-line-height: var(--space-4);
+  --ids-input-border-radius: var(--border-radius-sm);
+  --ids-input-border-color: var(--grey-300);
+  --ids-input-background: var(--samoyed);
+  --ids-input-margin: var(--space-1) var(--space-0);
+
+  /* Compact */
+  --ids-input-height-compact: var(--space-5);
+  --ids-input-font-size-compact: var(--font-size-3);
+  --ids-input-line-height-compact: var(--line-height-xxl);
+  --ids-input-letter-spacing-compact: var(--font-weight-regular);
+
+  /* Placeholder */
+  --ids-input-color-placeholder: var(--grey-400);
+
+  /* Hover */
+  --ids-input-border-color-hover: var(--grey-500);
+
+  /* Focus */
+  --ids-input-border-color-focus: var(--electric-blue-600);
+  --ids-input-border-radius-focus: var(--border-radius-md);
+  --ids-input-box-shadow-inner-color-focus: var(--samoyed);
+  --ids-input-box-shadow-outer-color-focus: var(--grey-300);
```

Change la valeur pour la couleur du focus
```suggestion
--ids-input-box-shadow-outer-color-focus: var(--electric-blue-600);
```